gems/aws-sdk-core/lib/aws-sdk-core/cbor/engines/default_cbor_engine.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative '../half'
```

Comment:
This seems like a slightly weird file structure now. Maybe we should have a module for our engine and have these files there?

Reply:
I removed half. But I think it's fine to require relative the encoder and decoder if the engines will live in a subfolder?